docs/Centaur.md

```diff
@@ -8,7 +8,7 @@ Centaur expects to find a Cromwell server properly configured and running in ser
 
 There are two ways to invoke the integration tests:
 
-* `sbt test` - compiles and run via sbt directly, simple but also has the problem of running two times cores tests in parallel which can overwhelm your Cromwell server if running in a development environment
+* `sbt test` - compiles and run via sbt directly, simple but also has the problem of running 2x cores tests in parallel which can overwhelm your Cromwell server if running in a development environment
```

Comment:
Not your fault :)

... but can we change `sbt test` to `sbt "centaur/it:test"`?